data/clients.json

```diff
@@ -597,7 +599,7 @@
 			"library": "Python",
 			"comments": "(Twisted client for Python 2 / 3)"
 		},
-		
+
```

Comment:
Looks like you accidentally added an extra line here. Could you remove it please?

```suggestion
```